packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -131,7 +131,11 @@ export type CustomCallControlButtonPlacement =
   | 'afterMicrophoneButton'
   | 'afterOptionsButton'
   | 'afterParticipantsButton'
-  | 'afterScreenShareButton';
+  | 'afterScreenShareButton'
+  // Sidepane placements
+  | 'beforeChatButton'
+  | 'afterChatButton'
+  | 'afterPeoplesButton';
```

Comment:
nit peoples -> people
```suggestion
  | 'afterPeopleButton';
```